<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>304</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>182</width>
    <height>304</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>182</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select</string>
  </property>
  <widget class="QPushButton" name="atBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>253</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Attach</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detBtn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>253</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Detach</string>
   </property>
  </widget>
  <widget class="QPushButton" name="moveBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>43</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eachBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Each</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allBtn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>43</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fngBtn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>83</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fingers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fromBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toBtn">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>170</y>
     <width>80</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>98</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:+:  Selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>157</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:+:  Translation by Pivot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>153</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:+:  Simple Constrain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBtn">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
